src/cascade/executor/epiviz_runner.py

```diff
@@ -49,37 +61,73 @@ def execution_context_from_settings(settings):
         model_version_id=settings.model.model_version_id,
         model_title=settings.model.title,
         gbd_round_id=settings.gbd_round_id,
+        bundle_id=settings.model.bundle_id,
+        add_csmr_cause=settings.model.add_csmr_cause,
+        location_id=settings.model.drill_location,
+    )
+
+
+def meas_bounds_to_stdev(df):
+    df["standard_error"] = (df.meas_upper - df.meas_lower) / (2 * 1.96)
+    df["standard_error"] = df.standard_error.replace({0: 1e-9})
+    df = df.rename(columns={"meas_value": "mean"})
+    df["density"] = DensityEnum.gaussian
+    df["weight"] = "constant"
+    return df.drop(["meas_lower", "meas_upper"], axis=1)
+
+
+def add_mortality_data(model_context, execution_context):
+    csmr = meas_bounds_to_stdev(
```

Comment:
Do we add mortality as an observation, or is the "add mortality data" drop-down really about setting mtother from that particular mortality? Maybe that drop-down means to do both, add the measure and add mtother. I need to look at DismodODE.

Reply:
I believe there are rules that ODE and to some extent Greg use when deciding what mortality data to add and/or synthesize under what circumstances. That's a thing I don't fully understand yet.